devtools-ci-index.yaml

```diff
@@ -2134,6 +2165,24 @@
             timeout: '30m'
         - '{ci_project}-{git_repo}-fabric8-analytics-pylint':
             git_repo: fabric8-analytics-server
+        - '{ci_project}-{git_repo}-fabric8-hdd':
+            git_organization: fabric8-hdd
+            git_repo: supervisor
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_run_tests.sh'
+            timeout: '30m'
+            image_name: 'fabric8-hdd/openshift-hdd-supervisor'
+        - '{ci_project}-{git_repo}-build-master':
+            git_organization: fabric8-hdd
+            git_repo: supervisor
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
+            saas_git: saas-hdd # To Be created
```

Comment:
If we merge this PR before saas-hdd is created, this will result in a failed build. But that should be fine to start with.

Reply:
Ack